src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Address.kt

```diff
@@ -26,7 +26,18 @@ import javax.persistence.Table
 @Table(name = "ADDRESSES")
 @DiscriminatorColumn(name = "OWNER_CLASS")
 @Inheritance
-abstract class Address {
+abstract class Address(
+  val premise: String? = null,
+  val street: String? = null,
+  val locality: String? = null,
+  @Column(name = "START_DATE")
+  val startDate: LocalDate = LocalDate.now(),
+  @Column(name = "NO_FIXED_ADDRESS_FLAG")
+  val noFixedAddressFlag: String = "N",
+  @OneToMany(mappedBy = "address", cascade = [CascadeType.ALL], fetch = FetchType.LAZY, orphanRemoval = true)
+  @Where(clause = "OWNER_CLASS = '" + AddressPhone.PHONE_TYPE + "'")
```

Comment:
would be better to do string interpolation here rather than adding 3 strings together